apps/src/util/experiments.js

```diff
@@ -65,6 +65,8 @@ experiments.FREE_RESPONSE_AI_ANALYSIS = 'free_response_ai_analysis';
 experiments.TEACHER_HOMEPAGE_V2 = 'teacher-homepage-v2';
 // Use the new version of codebridge share view
 experiments.CODEBRIDGE_SHARE = 'codebridge-share';
+// Allows teachers to use the ai enabled csp rubrics
+experiments.CSP_RUBRICS_ENABLED = 'csp-rubrics-enabled';
```

Comment:
If you go the route of changing the script_levels_controller so we have info about whether a rubric is experimental or not, we could switch this to something more generics like `experimental-rubrics-enabled` and then use it for any rubric we want to do a slow release for.  (This is assuming we're planning to launch more rubrics over time 🤷‍♀️ ). I don't feel strongly about it and can see arguments for keeping the flag very specific too.